recipe_scrapers/_abstract.py

```diff
@@ -1,15 +1,14 @@
 from collections import OrderedDict
 from json.decoder import JSONDecodeError
+from typing import Any, Dict, List, Optional, Tuple, Union
```

Comment:
If it's not a massive pain, what do you think about splitting these multi-reference imports into multi-line import format?

For example:

```py
from typing import (
    Any,
    Dict,
    List,
    Optional,
    Tuple,
    Union,
)
```

That's based on some ideas I picked up from https://github.com/asottile/reorder_python_imports about reducing conflicts.

(personally I prefer the look of references-within-parens to reduce repetition of the `typing` module reference, but I understand why having standalone import lines makes sense, too)

Reply:
I'm going to politely say that this is topic for another PR and issue. I have no preference here, but I made #332 just to force consistency and catch silly errors. Feel free to change it how you see fit.

Reply:
Cool, yep - makes sense.  I might have a look into opening a PR for this soon.